pendulum:
    start_cartx: 0
    start_angle: 1
    track_length: 1.0
    edge: 0.5
    gui_update_frequency: 20
    simulation:
        realtime_multiplier: 1
        cart_friction: 0.08
        delta_time: 0.01
        gravity: 9.81
        pole_friction: 0.002
        #mass: 0.0335
        mass: 0.1
        pole_length: 0.2
    pid:
        update_frequency: 10
        angle:
            p: 4
            i: 0.0
            d: 1
        cart:
            p: -10
            i: 0.0
            d: 10
